src/components/CartProduct.js

```diff
@@ -0,0 +1,62 @@
+import React, { useContext } from "react";
+import { FaTimes } from "react-icons/fa";
+
+//Context
+import CartContext from "../state/CartContext";
+
+//CSS
+import "../styles/Product.css";
+
+const CartProduct = ({ product }) => {
+  const { productsOnCart, setProductsOnCart } = useContext(CartContext);
+
+  const RemoveFromCart = () => {
+    let updatedCart = productsOnCart.filter(
+      (cartItem) => cartItem.id !== product.id
+    );
+    setProductsOnCart(updatedCart);
+  };
+
+  const UpdateItem = (event) => 
+  {
```

Comment:
I notice we can have an article in the cart wit quantity 0, we can fix this by:

```suggestion
{
if(event.target.value*1 === 0 ){
      RemoveFromCart()
      return 
    }
```

or set the `min` property of the input to 1